include/fmt/core.h

```diff
@@ -580,7 +580,7 @@ FMT_MAKE_VALUE(FMT_POINTER, std::nullptr_t, const void*)
 // formatting of "[const] volatile char *" which is printed as bool by
 // iostreams.
 template <typename T>
-void make_value(const T *p) {
+void make_value(const T */*p*/) {
```

Comment:
I suggest just removing `p`. Also "decuction" -> "deduction" in the commit title.